### PR TITLE
Patch the HPO download resource correctly in a test

### PR DESCRIPTION
**Pull request by northwestwitch**
Fixing a test failing when the real resource is not available. Real resource: http://compbio.charite.de/jenkins/job/hpo.annotations/lastSuccessfulBuild/artifact/util/annotation/genes_to_phenotype.txt

**How to test**:
1. Pytest should pass

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Hopefully this should fix those tests @mhkc

File: tests/commands/download/test_download_hpo_cmd.py [hunk 1]
```diff
@@ -4,19 +4,22 @@
...
 
 
 def test_download_hpo_cmd(mocker, empty_mock_app):
     """Test download hpo command"""
+
     # GIVEN a temporary directory
     mock_app = empty_mock_app
     runner = mock_app.test_cli_runner()
 
-    mocker.patch("scout.commands.download.hpo.print_hpo")
```
Comment (northwestwitch) [inline tests/commands/download/test_download_hpo_cmd.py:15]: The previous patch was targeting a function called later than fetch_resource (that is getting the resource from internet)

**Review by dnil — APPROVED** [review]

**Comment by dnil** [comment]
Actually the real resource moved, but that can happen again, or it can be down or blocked, so good to have this in place anyway. 👍

**Review by mhkc — APPROVED** [review]
Looks good

**Pull request merged**